src/MCPClient/lib/clientScripts/archivematicaCreateMETSMetadataCSV.py

```diff
@@ -43,7 +51,7 @@ def parseMetadata(SIPPath):
     See parseMetadataCSV for details on parsing.
 
     :param SIPPath: Path to the SIP
-    :return: {<filename>: OrderedDict{<metadata name>=<metadata value>} }
+    :return: {<filename>: OrderedDict(key: [values]) }
```

Comment:
I mostly used the OrderedListsDict so I didn't have to handle duplicates myself when inserting. Should I ```return OrderedDict(metadata)``` instead?